### ISSUE TITLE
Diagnostic updates cause WY failures in branch with parallel fixes

### ISSUE THREAD

**Issue by jeff-cohere** [TDycores-Project/TDycore#224]
Some recent updates have cause failures in the WY demos. @bishtgautam submitted PR #221, which applies several fixes to restore the dycore's parallel capabilities. Then PR #220 was merged, adding an interface for diagnostic fields. In a [separate branch](https://github.com/TDycores-Project/TDycore/blob/bishtgautam/fix-parallel-runs-rebase-02-02-2022/) that contains the changes for #221 rebased to `master` post #220, the WY demos stopped working. Gautam's assessment:

> The transient WY tests are failing. The residual for some cells is garbage at [line 1167 of tdywy.c](https://github.com/TDycores-Project/TDycore/blob/bishtgautam/fix-parallel-runs-rebase-02-02-2022/src/fe/tdywy.c#L1167) in the rebased branch. But, I can’t figure out where I made the mistake.

We're tracking that issue here so we can go ahead and merge #221.

We should probably add more parallel tests to our regression suite to make sure we don't damage the parallel runs in our future work.

**Comment by jeff-cohere** [comment]
Hey @bishtgautam , 

I'm coming up for air after a crazy month

**Comment by jeff-cohere** [comment]
Hey @bishtgautam , can you show me how to reproduce this issue? I'm wondering if it still occurs, and/or how much we should focus on fixing the WY tests when we know that discretization is going to be reworked.

**Comment by nocollier** [comment]
I thought we had discussed just removing it as needed? We can always
reconstruct it later if it is needed.

**Comment by jeff-cohere** [comment]
Yeah, maybe we should just "comment out" these regression tests for now. Thanks, @nocollier .
